superset-frontend/src/dashboard/components/menu/WithPopoverMenu.tsx

```diff
@@ -41,6 +42,67 @@ interface WithPopoverMenuState {
   isFocused: Boolean;
 }
 
+const WithPopoverMenuStyles = styled.div`
+  ${({ theme }) => css`
+    position: relative;
+    outline: none;
+
+    &.with-popover-menu--focused:after {
+      content: '';
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      border: 2px solid ${theme.colors.primary.base};
+      pointer-events: none;
+    }
+
+    .dashboard-component-tabs li &.with-popover-menu--focused:after {
+      top: ${theme.gridUnit * -3}px;
+      left: ${theme.gridUnit * -2}px;
+      width: calc(100% + ${theme.gridUnit * 4}px);
+      height: calc(100% + ${theme.gridUnit * 7}px);
+    }
+  `}
+`;
+
+const PopoverMenuStyles = styled.div`
+  ${({ theme }) => css`
+    position: absolute;
+    flex-wrap: nowrap;
+    left: 1px;
+    top: -42px;
```

Comment:
Not sure if these should be in gridUnits, or if it's -42 for a specific reason.

Reply:
Yes it should be in gridUnits. It's related to component's height (40px or 10gridUnits)